### ISSUE TITLE
InteractionModel Validation - missing intent

### ISSUE THREAD

**Issue by jkelvie** [bespoken/bst#95]
From the Developer console:
Error: There was a problem with your request: The intent 'LatestAudio' was not found in the domain definition. Occurred in sample 'LatestAudio latest' on line 7. 

This was for a case where an intent was in the sample utterances but not in the intent list.

**Comment by OpenDog** [comment]
This is normal. It's parent/child relationship.

**Comment by jkelvie** [comment]
Right, just noting it as a validation we can include in our system - I got
it on the Alexa developer console today.

We currently "okay" a file structured in this manner, but Amazon does not.

-John
